specifyweb/settings/specify_settings.py

```diff
@@ -1,8 +1,9 @@
+import os
 
 # Specify 7 requires the files from a Specify 6 install.
 # This setting should point to a directory containing an installation
 # of Specify 6 of the same version as the Specify database.
-THICK_CLIENT_LOCATION = '/opt/Specify'
+THICK_CLIENT_LOCATION = os.environ.get('THICK_CLIENT_LOCATION', '/opt/Specify')
```

Comment:
This allows local installations to set a THICK_CLIENT_LOCATION environment variable and use that instead of modifying this variable directly. 

Do we want to add note of this to the installation instructions? 
https://github.com/specify/specify7/blob/18f5c15a8e54d945b2c03a7877d453c8e1ea5669/README.md?plain=1#L491-L493

Reply:
And for Dockerized instances, do we want to provide documentation for how this environment variable can be used?
Or maybe just how it's used with correlation to the `docker-compose.yml` (and `Dockerfile`)? 

For the users who wish to go in depth for customizing the setup of their instance, there's a LOT of moving parts. They might not even know what's possible/not-possible, and I could see how easy it might be to get stuck.

Reply:
1. I don't think we should encourage anyone to configure new local installations
2. Documentation on supported environment variables are always desirable 😄